tests/openebs-pool-pod-failure_test.go

```diff
@@ -166,4 +192,97 @@ var _ = Describe("BDD of openebs pool pod failure experiment", func() {
 		})
 	})
 
+	//Matching the Resource Verison after Chaos
+	Context("Check Resource Version of pool container", func() {
+
+		It("Should check for the change in Resource Version after Chaos", func() {
+
+			resourceVersionSumAfter := 0
+			csp_rv, err := client.CoreV1().Pods(cspPodNs).List(metav1.ListOptions{LabelSelector: cspPodLabels})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, podSpec := range csp_rv.Items {
+				rv, _ := strconv.Atoi(podSpec.ResourceVersion)
+				resourceVersionSumAfter = resourceVersionSumAfter + rv
+			}
+
+			Expect(resourceVersionSumBefore-resourceVersionSumAfter).NotTo(Equal(0), "The Resource Version does not change")
+			fmt.Printf("The Resource Version changes\n")
+
+		})
+	})
+
+	//Matching the Pool Pod Name
+	Context("Check csp pod Name", func() {
+
+		It("Should check for the change in name of csp pod after Chaos", func() {
+
+			var podNameAfter [3]string
+			podNameChanged := false
+			csp_name, err := client.CoreV1().Pods(cspPodNs).List(metav1.ListOptions{LabelSelector: cspPodLabels})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for i, podSpec := range csp_name.Items {
+				podNameAfter[i] = podSpec.Name
+			}
+
+			Expect(podNameChanged).NotTo(Equal("false"), "The csp pod name does not change")
+			fmt.Printf("CSP pod name Changes!!!\n")
+
+		})
+	})
+
+	//Matching csp pod PodIP
+	Context("Check csp pod PodIP", func() {
+
+		It("Should check for the change in csp pod PodIP after Chaos", func() {
+
+			var podIpAfter [3]string
+			podIpChanged := false
+			csp_podip, err := client.CoreV1().Pods(cspPodNs).List(metav1.ListOptions{LabelSelector: cspPodLabels})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for i, podSpec := range csp_podip.Items {
+				podIpAfter[i] = podSpec.Status.PodIP
+			}
+
+			Expect(podIpChanged).NotTo(Equal("false"), "The csp pod PodIP does not change")
```

Comment:
same here, Pls add the matching of podIpBefore & podIpAfter and verify that both are different,